# Add missing tspi_freeTable function signature to fix compiler warnings.
# Suitable for use upstream.
# This change was implemented in-house.
#
--- src/include/tsp_delegate.h	2010-03-11 12:26:51.000000000 -0800
+++ src/include/tsp_delegate.h	2012-04-23 14:37:24.040135000 -0700
@@ -19,5 +19,6 @@
 TSS_RESULT	update_delfamily_object(TSS_HTPM, UINT32);
 TSS_RESULT	get_delegate_index(TSS_HCONTEXT, UINT32, TPM_DELEGATE_PUBLIC *);
 TSS_RESULT	__tspi_build_delegate_public_info(BYTE, TSS_HPCRS, TSS_HDELFAMILY, TSS_HPOLICY, UINT32 *, BYTE **);
+TSS_RESULT	__tspi_freeTable(TSS_HCONTEXT tspContext);
 
 #endif
